spec/models/taxon_name_spec.rb

```diff
@@ -79,6 +79,28 @@
     expect( TaxonName.make!( lexicon: " Foo" ).lexicon ).to eq "Foo"
   end
 
+  it "should not be valid with a non-English translation of a lexicon" do
+    tn = TaxonName.make(lexicon: "简体中文", name: "common")
+    expect(tn).to_not be_valid
+    expect(tn.errors.messages[:lexicon].count).to eq 2
+  end
+  
+  it "should parameterize and store lexicon" do
+    tn = TaxonName.make!(lexicon: TaxonName::LEXICONS[:CHINESE_SIMPLIFIED], name: "common")
+    expect(tn.parameterized_lexicon).to eq "chinese-simplified"
+  end
+  
+  it "should not parameterize and store lexicon with invalid characters" do
+    tn = TaxonName.make(lexicon: "测试", name: "common")
+    expect(tn).to_not be_valid
+    expect(tn.errors.messages[:lexicon].count).to eq 1
```

Comment:
Same as above but for `expect(tn.errors.messages[:lexicon]).to include(t(:should_be_english))`